oneapp-server/types/Location.js

```diff
@@ -0,0 +1,39 @@
+const { gql } = require('apollo-server-express');
+const { allow } = require('graphql-shield');
+
+const typeDef = gql`
+  extend type Query {
+    "All counties"
+    counties: [Location!]
+    "Check if zipcode and county provided is a valid NJ zipcode"
+    isValidNJZipcode(ZIPCODE: Int!, COUNTY_NUMBER: Int!, IS_HOMELESS: Boolean!): Boolean
+    "Returns county name of given user"
+    getCountyDetails: Location
+  }
+
+  type Location {
+    COUNTY_NAME: String,
+    COUNTY_NUMBER: Int,
+  }
+`;
+
+const resolvers = {
+  Query: {
+    counties: (_parent, _args, { dataSources }) => dataSources.LocationDao.getCounties(),
+    isValidNJZipcode: async (_parent, { ZIPCODE, COUNTY_NUMBER, IS_HOMELESS }, { dataSources }) => {
+      const response = await dataSources.LocationDao.isValidNJZipcode(ZIPCODE, COUNTY_NUMBER, IS_HOMELESS);
+      return response[0];
```

Comment:
I think it would make sense here to to have the Dao parse the response value and return a boolean. There are a couple of other Dao methods where I have returned the raw response to the resolver (like in userPasswordReset), but that was because the stored procedure had multiple parameters binded to the output (in that case, it was returning both the user object and a response code. Thoughts?